framework/src/bundle/CoreBundleContext.cpp

```diff
@@ -72,6 +76,10 @@ std::unordered_map<std::string, Any> InitProperties(
     std::string(CppMicroServices_VERSION_STR);
   configuration[Constants::FRAMEWORK_VENDOR] = std::string("CppMicroServices");
 
+#ifdef US_PLATFORM_POSIX
+  configuration[Constants::LIBRARY_LOAD_OPTIONS] = Any(RTLD_LAZY | RTLD_LOCAL);
```

Comment:
Isn't insert the right API to use here? Looks like this overrides user provided value

Reply:
Done.